### PR TITLE
fix(readme): rework guidelines and add limitation for images

### PR DESCRIPTION
**Pull request by tnguyen1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,133 @@
...
+A livereload is also set up and allows any change to the `md` files to reload its html version in the browser.
+
 
 ## Contribute
 
+### Update existing pages
+
+In this case, a simple edit of the _.md_ file in the appropriate branch (one per version) is enough, the build process will do the rest.
+
+<u>Example</u>: To edit the **actors.html** page in the 7.3 version, checkout the **versions/7.3** branch and edit the `md/actors.md` file.
```
Comment (redboul) [inline README.md]: currently, there is no version per branch and we still did not clarify this...
The way it is right now is that the `master` branch contains 7.3 and the `dev` branch contains 7.4.

File: README.md [hunk 2]
```diff
@@ -1,37 +1,133 @@
...
+
+### Add new pages
+
+#### Create new pages
+
+Checkout the version you want this page to be available in and create the _.md_ file in the **md** folder. The **master** branch contains documentation applicable to the latest version of Bonita BPM.
```
Comment (redboul) [inline README.md]: same remark than above

File: README.md (hunk 1 above)
Comment (tnguyen1) [inline README.md]: Agreed - should we create a `versions/7.3` branch now that 7.3 is released?
Or change to follow the development branch model with master/dev ?

Comment (redboul) [inline README.md]: A dev branch has been created but I do not think that this model can be applied here.
We should create a 7.3 branch and have the master branch pointing to the next version.
What do you think?

Comment (tnguyen1) [inline README.md]: OK with creating a 7.3 branch and master pointing to 7.4.
Then how to deal with changes applicable to both branches? Shall the author create 2 pull requests (one on 7.3 branch, one on master branch)? If so, we need to communicate the process I guess.

Comment (redboul) [inline README.md]: we should proceed the same way than with development project : rebase master branch on previous branch

Comment (tnguyen1) [inline README.md]: I'm not sure we can rebase master branch on previous version branches.
I think each version branch is likely to have its own lifecycle.

Basically we need to consider these cases (given 7.3 and 7.4=current version):
- update on 7.3 only
- update on 7.4 only
- update on both 7.3 and 7.4

Better discuss about it when you are available.

Comment (tnguyen1) [inline README.md]: Up / Are we clear on how we manager versions/branches?

**Pull request closed without merging**